CorsixTH/Lua/world.lua

```diff
@@ -1305,15 +1305,17 @@ function World:getCampaignWinningText(player_no)
         end
       end
     end
-    local level_info = TheApp:readLevelFile(self.map.level_number)
     text[1] = _S.letter.dear_player:format(self.hospitals[player_no].name)
     if has_next then
-      text[2] = level_info.end_praise and level_info.end_praise:format(next_level_name) or _S.letter.campaign_level_completed:format(next_level_name)
-      text[3] = ""
+      if next_level_info.end_praise then
+        text[2] = next_level_info.end_praise:format(next_level_name)
+      else
+        text[2] = _S.letter.campaign_level_completed:format(next_level_name)
+      end
```

Comment:
Not your fault, but `next_level_name` isn't initalized:

![image](https://github.com/user-attachments/assets/8c748987-2063-4e7b-b39a-0286688c084b)


Keeping the declaration close to first use makes this much simpler to detect.

Reply:
It's declared out of the loop, and set inside in on L1302.

Reply:
Ah ok, my bad